src/api/models/employers.js

```diff
@@ -9,12 +9,18 @@ var employerProfileSchema = new mongoose.Schema({
     links: {
         type: [String]
     },
-    location: {
-        type: [Number]
+    locations: {
+        type: [String]
     },
     attendance_date: {
         type: Date
     },
+    map_coordinates: {
```

Comment:
why is this not just latitude longitude? Is this for the career fair map?

```
map_coordinate: {
    latitude: {
        type: Number
    },
    longitude: {
        type: Number
    }
}
```

Reply:
I left it flexible for future proofing since some fairs may only be on 1 floor. But CoC's fair is usually on 3 floors, so in that case we need 3 coordinates: x/y for a given floor, and z to specify which floor.

Reply:
Other fairs could theoretically spread out across not just multiple floors, but multiple buildings too. So a 4th coordinate would be needed for those